### PR TITLE
Make icons of footer not to overflow:

### PR DESCRIPTION
**Pull request by mani87**
Icons of the footer overflows when
screen size is reduced. This is
improved with change in css file.

Closes https://github.com/fossasia/fossasia.org/issues/194

### CODE REVIEW

**Comment by mani87** [comment]
screenshot_20171012_232138

**Comment by mani87** [comment]
@isuruAb @geekyd Can you please review this?
Thankyou :)

**Review by isuruAb — COMMENTED** [review]
@mani87  please remove the dark gray column which is in right side of the screen

**Comment by mani87** [comment]
@isuruAb Done! Can you review this please?
Thankyou :)
screenshot_20171015_181732

**Comment by isuruAb** [comment]
hello @mani87 but there is a horizontal scroll as I see. please check it. please add a preview link to your PR. then reviewers can review it easily.

**Comment by mani87** [comment]
@isuruAb hey! Since the size of icons and distance between them cannot be changed that is why this happens in certain devices. Can you give me little help?
The picture below is the new screenshot after removing horizontal scrolling. Can you tell me is it right, so that i can commit and push changes. Thankyou!
screenshot_20171015_194711

**Comment by isuruAb** [comment]
@mani87  yes last one seems good.please provide a preview link with your PR. Then it is even easy to review. cheers 👍

**Comment by mani87** [comment]
@isuruAb here it is.
https://rawgit.com/mani87/fossasia.org/footer_alignment/index.html

**Review by dilpreetsio — CHANGES_REQUESTED** [review]
@mani87 Please check the screen shot seems like the width of the page is not set to 100%, screen shot taken on a 1080p display
screenshot from 2017-10-16 20-12-15

File: index.html [hunk 1]
```diff
@@ -1020,7 +1022,8 @@ <h1 class="large-h1 text-white">Subscribe to stay informed</h1>
 							</ul>
 						</div>
 					</div>
-
+					<center>
```
Comment (dilpreetsio) [inline index.html:1025]: Please do not use `center` tag instead implement it using CSS.

Comment (mani87) [reply index.html:1025]: @geekyd can you tell me if its right now?
screenshot_20171025_232617

Comment (mani87) [reply index.html:1025]: Screen size is 1920*1080.

**Comment by mariobehling** [comment]
Closing due to inactivity.

**Pull request closed without merging**